config/jobs/kubernetes-sigs/testgrid/testgrid-presubmits.yaml

```diff
@@ -7,12 +7,17 @@ presubmits:
     spec:
       containers:
       # See https://playwright.dev/docs/ci
-      - image: mcr.microsoft.com/playwright:v1.37.0-focal
+      - image: mcr.microsoft.com/playwright:v1.46.0-noble
```

Comment:
This may breaks tests. Starting with [1.38](https://github.com/microsoft/playwright/releases/tag/v1.38.0):

> Breaking Changes: Playwright no longer downloads browsers automatically

We should probably add `playwright/test` as a follow-up.

Reply:
Gotcha, thanks for catching!

Reply:
Ah, I think it should be fine? We're actually using `@web/test-runner-playwright`, and I think it's just using playwright directly (which doesn't seem recommended anymore but should work: https://github.com/modernweb-dev/web/blob/master/packages/test-runner-playwright/package.json)